set.mm

```diff
@@ -464521,6 +464601,17 @@ Class abstractions (a.k.a. class builders)
       ( vy cv wcel wa csn cxp ciun cima cop vex elimasn wb opeliunxp baib eqrdv
       adantl syl5bb ) ABJZCKZLZIBCUFMZDNOZUIPZDIJZUKKUFULQUJKZUHULDKZUJUFULBRIR
       SUGUMUNTAUMUGUNBCDULUAUBUDUEUC $.
+
+    $d A x z $.  $d B z $.  $d C z $.  $d ph z $.  $d x y z $.
+    iunsnima2.1 $e |- F/_ x C $.
+    iunsnima2.2 $e |- ( x = y -> B = C ) $.
+    $( Version of ~ iunsnima with different variables.  (Contributed by Thierry
+       Arnoux, 22-Jun-2024.) $)
+    iunsnima2 $p |- ( ( ph /\ y e. A ) -> ( U_ x e. A ( { x } X. B ) " { y } )
+       = C ) $=
```

Comment:
I think the setvar variable `y` can be replaced by a class variable `Y`, since `Y` is actually a set because of `Y e. A`.

Reply:
You're right! I'll do the change.